src/app/shared/message-table/value.pipe.ts

```diff
@@ -0,0 +1,13 @@
+import { Pipe, PipeTransform } from '@angular/core';
+
+import { MainfluxMsg } from 'app/common/interfaces/mainflux.interface';
+
+@Pipe({name: 'valuePipe'})
```

Comment:
Simply `value`?

Reply:
It seemed to me that name would be prone to conflict, but since it's pipe, probably it's ok.